lm_eval/tasks/gpqa/zeroshot/utils.py

```diff
@@ -0,0 +1,28 @@
+import datasets
+import re
+
+
+def preprocess(text):
+    if text is None:
+        return " "
+    text = text.strip()
+    text = text.replace(" [title]", ". ")
+    text = re.sub("\\[.*?\\]", "", text)
+    text = text.replace("  ", " ")
+    return text
+
+
+def process_docs(dataset: datasets.Dataset) -> datasets.Dataset:
+    def _process_doc(doc):
+        # breakpoint()
+        out_doc = {
+            # "query": "Question: " + preprocess(doc["instruction"]) + "\nAnswer:",
+            "choice1": preprocess(doc["Incorrect Answer 1"]),
+            "choice2": preprocess(doc["Incorrect Answer 2"]),
+            "choice3": preprocess(doc["Incorrect Answer 3"]),
+            "choice4": preprocess(doc["Correct Answer"]),
```

Comment:
This currently has the correct answer being always (D) , which will make the task artificially easy especially in the few-shot setting.

Reply:
Thank you for your suggestion! I've made the suggested modifications to address the issues you raised.

Reply:
Perfect, thank you!